creator-node/src/models/index.js

```diff
@@ -18,6 +20,23 @@ const sequelize = new Sequelize(config.get('dbUrl'), {
     min: 5,
     acquire: 60000,
     idle: 10000
+  },
+  dialectOptions: {
+    // number of milliseconds before a statement in query will time out, default is no timeout
+    statement_timeout: STATEMENT_TIMEOUT,
```

Comment:
i think we can enable statement_timeout and idle_in_transaction_session_timeout and set them both to the config above. query_timeout seems to be something from the postgres client but we want the db to kill the query on it's side to free up resources. also connectionTimeoutMillis seems to be with connection acquisition but we have a connection pool setting that does that. https://github.com/AudiusProject/audius-protocol/blob/master/creator-node/src/models/index.js#L19

Reply:
also here's the link to postgres with the actual docs for each of these https://www.postgresql.org/docs/current/runtime-config-client.html#GUC-STATEMENT-TIMEOUT

Reply:
this option only exists in latest sequelize
https://github.com/sequelize/sequelize/blob/main/src/dialects/postgres/connection-manager.js

our version only has the `statement_timeout` option so i'm just gonna set that for now

also our connection pool specifies an idle value so i think that might be sufficient?